### DISCUSSION TITLE
OAuth Solutions?

### DISCUSSION THREAD

**Discussion by gavindoughtie** [onejs/one#149] [Q&A] [answered]
Playing with the full stack example and (boo-hoo) no guidance on OAuth solutions that fit well with the onejs philosophy. Any suggestions? Looking to get the auth part over with with minimal headaches. NextAuth?

**Comment by natew** [comment, marked as answer]
We have an even fuller stack example that isn't released yet, but shows how to do `@supabase/ssr`:

https://github.com/onejs/one/blob/main/examples/one-fullstack/features/api/getSupabaseServerClient.ts

Also the tamagui.dev site source code, where we actually have it working in prod, so maybe more useful reference.
